src/components/Menu.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+import '../styles/menu.css'
+
+class Menu extends React.Component {
+  render() {
+    return (
+
+      <div className="account-menu">
+        <div className="account-menu-list">
+          <div className="account-menu-list-item"><a href="#">Account </a></div>
```

Comment:
I would omit whitespace within the tag (it could lead to subtle rendering errors on different browsers iirc)